object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -76,29 +60,8 @@
 
   private final boolean useValueChecker;
 
-  // The list is copied from lombok.core.handlers.HandlerUtil. The list cannot be used from that
-  // class directly because Lombok does not provide class files for its own implementation, to
-  // prevent itself from being accidentally added to clients' compile classpaths. This design
-  // decision means that it is impossible to depend directly on Lombok internals.
-  /** The list of annotations that Lombok treats as non-null. */
-  public static final List<String> NONNULL_ANNOTATIONS =
-      Collections.unmodifiableList(
-          Arrays.asList(
-              "android.annotation.NonNull",
-              "android.support.annotation.NonNull",
-              "com.sun.istack.internal.NotNull",
-              "edu.umd.cs.findbugs.annotations.NonNull",
-              "javax.annotation.Nonnull",
-              // "javax.validation.constraints.NotNull", // The field might contain a null value
-              // until it is persisted.
-              "lombok.NonNull",
-              "org.checkerframework.checker.nullness.qual.NonNull",
-              "org.eclipse.jdt.annotation.NonNull",
-              "org.eclipse.jgit.annotations.NonNull",
-              "org.jetbrains.annotations.NotNull",
-              "org.jmlspecs.annotation.NonNull",
-              "org.netbeans.api.annotations.common.NonNull",
-              "org.springframework.lang.NonNull"));
+  /** The collection of built-in framework support for object construction checker */
```

Comment:
Minor comments about grammar:
* should be "for *the* object construction checker"
* it's usually better style to end comments with a period (just like a sentence)